src/hotspot/cpu/riscv/gc/x/x_riscv.ad

```diff
@@ -52,11 +52,11 @@ static void x_load_barrier_slow_path(MacroAssembler& _masm, const MachNode* node
 %}
 
 // Load Pointer
-instruct xLoadP(iRegPNoSp dst, memory mem)
+instruct xLoadP(iRegPNoSp dst, memory mem, rFlagsReg cr)
```

Comment:
Add a tmpReg here, as t1 is also used by masm.
ppc have added tempregs for all these I think we should also do that as both t0 and t1 can be used by masm.